geode-assembly/src/test/java/org/apache/geode/rest/internal/web/controllers/RestAPIOnRegionFunctionExecutionDUnitTest.java

```diff
@@ -26,7 +26,9 @@
 import java.util.Map;
 import java.util.Set;
 
+import org.apache.geode.test.dunit.rules.RequiresGeodeHome;
```

Comment:
Reorder imports - looks like there are several files that should have the imports reordered. I'd suggest reviewing all of the tests modified and use the IDE to reorder the imports.